benchmark-base/src/main/java/org/conscrypt/BenchmarkProtocol.java

```diff
@@ -0,0 +1,18 @@
+package org.conscrypt;
+
+@SuppressWarnings("ImmutableEnumChecker")
+public enum BenchmarkProtocol {
+
+    TLSv13_ONLY("TLSv1.3"),
```

Comment:
Why "Only"? Why not just `TLSv13`?

Reply:
Oh, I was originally going to have a TLSv12and13 or something along those lines, but ended up scrapping it because it didn't really make any difference, then forgot to go back and update the names.  Fixed.